### ISSUE TITLE
Wrong ambassadors are emailed for approval

### ISSUE THREAD

**Issue by oana-sipos** [codeeu/coding-events#443]
Norwegian ambassador is emailed for approving an event in Malta.
___
Hello, Øystein!

A new event Build your first Android Application – University of Malta has been added to the codeweek.eu site and it needs your revision. Would you be kind enough to check if everything is ok and approve it, or otherwise request an update: http://events.codeweek.eu/view/17737/build-your-first-android-application-university-of-malta/

Thank you for your help!

Codeweek.eu Team

**Comment by mitio** [comment]
The event is positioned in Norway, though. That's the problem. Someone wrote "Malta" and did not check where the marker was.

The event should be edited, but I think this is not an issue in the site itself.

**Comment by oana-sipos** [comment]
Indeed, I have also replied on the list and @tassos figured it out too. We have the address, so could we edit the event? Who is an admin and could do so? @alessandrobogliolo maybe?

**Comment by alessandrobogliolo** [comment]
Sometimes there are issues with geocoding. It depends on Google APIs. But I
don't know which are the triggering conditions.
Il 10/Ott/2016 19:48 "Dimitar Dimitrov" <notifications@github.com> ha
scritto:

**Comment by mitio** [comment]
@alessandrobogliolo has full admin access, he can edit the event, but I think that you should also have admin access (if not, Alessandro should probably add you as an admin since you're working on the site).
